ClusterCenters clusters
string label
geometry_msgs/Pose pose
int64 radius
int64 people
